books/free-programming-books-bl.md

```diff
@@ -104,6 +104,7 @@
 * [JavaScript Behind The Scene Bangla Tutorial](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDPaOrWvTIuhb5GRoJVWiE2) - Stack Learner
 * [JavaScript Full Playlist With Order](https://www.youtube.com/playlist?list=PL_XxuZqN0xVAJTV_1ZXwB1XIiFkK0ddZA) - Stack Learner
 * [Make Fun of JavaScript Array](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDr08QgQHljCecWtA4jBLnS) - Stack Learner
+* [JavaScript Bangla Tutorial](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9OkrURufHpGUUTBjJhO9Ghy) - Learn with Sumit Bangladesh
```

Comment:
Please, ensure fullfill author attributions @learnwithsumit

ℹ️ From his LinkedIn profile: `Sumit Saha`

⚠️ Needs real title: `JavaScript Bangla Tutorial | JS Bangla Tutorial Series for Beginners`